src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler_Helpers.cs

```diff
@@ -642,6 +643,11 @@ private static bool ShouldAddBraceForFinallyClause(FinallyClauseSyntax finallyCl
                && finallyClauseNode.Block.OpenBraceToken.IsMissing
                && !finallyClauseNode.Block.Span.Contains(caretPosition);
 
+        private static bool ShouldAddBraceForCheckedStatement(CheckedStatementSyntax checkedStatementNode, int caretPosition)
+            => checkedStatementNode.Block.OpenBraceToken.IsMissing
+                && checkedStatementNode.Block.Statements.IsEmpty()
+                && !checkedStatementNode.Block.Span.Contains(caretPosition);
```

Comment:
why the statements.isempty check?  i don't see that on any of hte others.

Reply:
: ) I will remove this one then.